src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  const BASE_PRICE = days * 40;
+
+  if (days < SHORT_TERM) {
+    return BASE_PRICE;
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
```

Comment:
Since the previous `if` statement contains a `return`, you don't need to check `days >= SHORT_TERM` again. If the code reaches this point, it's already guaranteed that `days` is at least `SHORT_TERM`.